operator/identitygc/gc.go

```
@@ -123,6 +123,16 @@ func registerGC(p params) {
 				return gc.startCRDModeGC(ctx)
 			case option.IdentityAllocationModeKVstore:
 				return gc.startKVStoreModeGC(ctx)
+			case option.IdentityAllocationModeDoubleWrite:
+				crdErr := gc.startCRDModeGC(ctx)
+				if crdErr != nil {
+					log.WithError(crdErr).Error("Unable to start CRD mode GC")
```

I would return the error for symmetry with the cases above. Same for the kvstore GC below.